src/framework/standard/args.rs

```diff
@@ -316,25 +483,12 @@ impl Args {
     ///
     /// assert_eq!(args.len_quoted(), 2); // `2` because `["42", "69"]`
     /// ```
+    #[deprecated(since = "0.5.3", note = "its task was merged with `len`, please use it instead.")]
```

Comment:
Should depreciation-notes start capitalised?

Reply:
Not really.
Though, i suppose i could.